### PR TITLE
feat(pacstall): spruce up help message

### PR DESCRIPTION
**Pull request by Henryws**
## Purpose

The current help message looks trashy and quickly made (it was).

## Approach

Make it look decent.

## Checklist

- [x] I confirm that I have read the [contributing guidelines](https://github.com/pacstall/pacstall/blob/develop/CONTRIBUTING.md), and this pull request is abiding by all the clauses stated in the guideline.

### CODE REVIEW

File: pacstall [hunk 1]
```diff
@@ -305,20 +305,39 @@ while [[ $1 != "--" ]]; do
...
--A --add-repo, Adds repo to your repo list
--U --update, Update script
--V --version, Prints pacstall version
--L --list, Lists installed packages
--Up --upgrade, Upgrades packages
--Qi --query-info, Get package info
--P --disable-prompts, Disable Prompts for unattended installation
--K --keep, retain build directory after package install"
+			echo -e "Usage: pacstall [-h] [-V] {-I,-Il,-S,-R,-D,-A,-U,-L,-Up,-Qi} [-P] [-K]
+
```
Comment (wizard-28) [inline pacstall:309]:
```suggestion
An AUR inspired package manager for Ubuntu.
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by D-Brox — APPROVED** [review]

**Review by volitank — APPROVED** [review]
Looks good to me. Force push master.

File: pacstall [hunk 2]
```diff
@@ -305,20 +305,41 @@ while [[ $1 != "--" ]]; do
...
+		Install a local pacscript.
+	${BOLD}-S${NC}, ${BOLD}--search${NC} <package>
+		Search for a package.
+	${BOLD}-R${NC}, ${BOLD}--remove${NC} <package>
+		Remove a package.
+	${BOLD}-D${NC}, ${BOLD}--download${NC} <package>
+		Download a pacscript.
+	${BOLD}-A${NC}, ${BOLD}--add-repo${NC} <repo>
+		Add a repository.
+	${BOLD}-U${NC}, ${BOLD}--update${NC}
```
Comment (wizard-28) [inline pacstall:325]: Shouldn't this have the syntax?

File: pacstall [hunk 3]
```diff
@@ -305,20 +305,41 @@ while [[ $1 != "--" ]]; do
...
--U --update, Update script
--V --version, Prints pacstall version
--L --list, Lists installed packages
--Up --upgrade, Upgrades packages
--Qi --query-info, Get package info
--P --disable-prompts, Disable Prompts for unattended installation
--K --keep, retain build directory after package install"
+			echo -e "Usage: pacstall [-h] [-V] {-I,-Il,-S,-R,-D,-A,-U,-L,-Up,-Qi} [-P] [-K]
+
+	An AUR inspired package manager for Ubuntu.
```
Comment (wizard-28) [inline pacstall:310]:
```suggestion
An AUR inspired package manager for Ubuntu.
```

File: pacstall [hunk 4]
```diff
@@ -305,20 +305,41 @@ while [[ $1 != "--" ]]; do
...
+	${BOLD}-Up${NC}, ${BOLD}--upgrade${NC}
+		Upgrade all installed packages.
+	${BOLD}-Qi${NC}, ${BOLD}--query-info${NC} <package>
+		Query information about a package.
+
+Options:
+	${BOLD}-P${NC}, ${BOLD}--disable-prompts${NC}
+		Disable prompts.
+	${BOLD}-K${NC}, ${BOLD}--keep${NC}
+		Keep the downloaded files.
```
Comment (wizard-28) [inline pacstall:338]:
```suggestion
		Keep the build files.
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: pacstall (hunk 2 above)
Comment (D-Brox) [reply pacstall:325]: Oh ye, the optional ones

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**